airbyte-integrations/connectors/source-airship/source_airship/spec.yaml

```diff
@@ -0,0 +1,20 @@
+documentationUrl: https://docs.airbyte.com/category/sources/airship
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Airship Spec
+  type: object
+  required:
+    - access_key
+  additionalProperties: true
+  properties:
+    access_key: 
```

Comment:
You can order the inputs (example [here](https://github.com/airbytehq/airbyte/pull/15880/files#diff-a8620b4d2a253892e3f1864be548549e6671e1ebcc348472165051761461e684R17))